cmd/juju/application/deploy.go

```diff
@@ -374,23 +374,28 @@ latter prefixed with "^", similar to the 'tags' constraint).
 
 
 Examples:
-    juju deploy mysql --to 23       (deploy to machine 23)
-    juju deploy mysql --to 24/lxd/3 (deploy to lxd container 3 on machine 24)
-    juju deploy mysql --to lxd:25   (deploy to a new lxd container on machine 25)
-    juju deploy mysql --to lxd      (deploy to a new lxd container on a new machine)
+    juju deploy mysql               (deploy to a new machine)
+    juju deploy mysql --to 23       (deploy to preexisting machine 23)
+    juju deploy mysql --to 24/lxd/3 (deploy to LXD container 3 on machine 24)
+    juju deploy mysql --to lxd:25   (deploy to a new LXD container on machine 25)
+    juju deploy mysql --to lxd      (deploy to a new LXD container on a new machine)
+
+    juju deploy mysql -n 2 --to 3,lxd:5
+    (deploy 2 units, one on machine 3 and one to a new LXD container
+    on machine 5)
+
+    juju deploy mysql -n 5 --constraints mem=8G
+    (deploy 5 units to machines with at least 8 GB of memory)
 
     juju deploy mysql --to zone=us-east-1a
     (provider-dependent; deploy to a specific AZ)
 
     juju deploy mysql --to host.maas
     (deploy to a specific MAAS node)
 
-    juju deploy mysql -n 5 --constraints mem=8G
-    (deploy 5 units to machines with at least 8 GB of memory)
-
     juju deploy haproxy -n 2 --constraints spaces=dmz,^cms,^database
-    (deploy 2 units to machines that are part of the 'dmz' space but not of the
-    'cmd' or the 'database' spaces)
+    (deploy 2 units to machines that are part of the 'dmz' space but
```

Comment:
"Part of" seems awkward here - what about just "in the 'dmz' space but not in the 'cmd' or 'database' spaces"?